### PR TITLE
General cleanup (needs testing)

### PR DESCRIPTION
**Pull request by hairychris**
The json package was imported twice.

### CODE REVIEW

**Comment by hairychris** [comment]
If you want to hold off on merging this I am going through having a bit of a clear up in places, might be good to get that all out of the way in one pull request.

File: tx_sign.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+"""Prepare transaction on a network connected machine."""
```
Comment (bargst) [inline tx_sign.py]: Would you update this header before I merge ? Anyway thanks for the PR, appreciated !

**Comment by hairychris** [comment]
Hey, I've done a little more cleanup. There is still some more to do (e.g. why is requests not used in push) but I think it should all be a bit easier to read now.

File: tx_sign.py (hunk 1 above)
Comment (hairychris) [inline tx_sign.py]: Fixed.

**Pull request merged**